pynbody/snapshot/ramses.py

```diff
@@ -794,7 +794,7 @@ def translate_info(sim):
 
     sim.properties['boxsize'] = sim._info['boxlen'] * l_unit
 
-    if sim._info['omega_k'] == sim._info['omega_l'] == sim._info['omega_b'] == 0.0:
```

Comment:
OK, weird, but I guess if that's true we have no option... is there a way to detect cosmological simulations done in OCDM (which would have omega_k = omega_l = 0)?